### PR TITLE
Update noejectdelay to 7.1.4

### PR DESCRIPTION
**Pull request by brianbrownton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by brianmorton** [comment]
Website seems outdated as it redirects to https://pqrs.org/osx/karabiner/ now... version was found by URL sniffing.

Maybe this cask should just be removed?

**Comment by commitay** [comment]
https://pqrs.org/osx/karabiner/noejectdelay.html only has `7.1.0`

Only works on el cap and older.

**Comment by brianmorton** [comment]
@commitay ahh ok... The URL in the cask is wrong then - it has a `.en` after the `.html` which is causing the redirect.

File: Casks/noejectdelay.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  sha256 'cfaeed50aa7ed9eac04716ab0e34c5ef10658802e691732f7589f9454e96aa94'
+  version '7.1.4'
+  sha256 '6a263c59efaaffb200db7d9ee6fe817f414e148053e610ccdcf37186fed27cc0'
 
   url "https://pqrs.org/osx/karabiner/files/NoEjectDelay-#{version}.dmg"
   appcast 'https://github.com/tekezo/NoEjectDelay/releases.atom',
-          checkpoint: '4475b83ad5927875fbc16e19c890dbaba1e466a758afcc0351ec86bb0e00468d'
+          checkpoint: '8cf6b036077bd9db8bd105ec548f60866403c2986e760dcc6d0c8b5b99916710'
   name 'NoEjectDelay'
   homepage 'https://pqrs.org/osx/karabiner/noejectdelay.html.en'
```
Comment (adidalal) [inline Casks/noejectdelay.rb]: `homepage 'https://pqrs.org/osx/karabiner/noejectdelay.html'`

File: Casks/noejectdelay.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
```
Comment (adidalal) [inline Casks/noejectdelay.rb]: Let's add a `depends_on macos: '<= :el_capitan'` and a `discontinued` caveat.

Example - https://github.com/caskroom/homebrew-cask/blob/f601dceb8d8a44d1aff6104daae3a2fcb0d6f32f/Casks/fraise.rb

**Review by adidalal — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Since we officially only support the latest macOS and the version before and this is already discontinued, might as well remove it.

https://github.com/caskroom/homebrew-cask/pull/43580

**Pull request closed without merging**